src/status_im/utils/config.cljs

```diff
@@ -20,12 +20,12 @@
 ;; flags stay up to date and are removed once behavior introduced is stable.
 
 (goog-define INFURA_TOKEN "800c641949d64d768a5070a1b0511938")
-(goog-define POKT_TOKEN "")
+(goog-define POKT_TOKEN "3ef2018191814b7e1009b8d9")
```

Comment:
Is it okay to publish the token? Or is it temporarily just for the sake of this PR build?

Reply:
I see the token was originally added in:

- https://github.com/status-im/status-mobile/pull/14144

And that token does not fit any of our currently used tokens that exist on [Grove City](https://portal.grove.city/) portal, so I don't think this token relevant anymore.

But anyway, we shouldn't be putting any tokens in plaintext in the repo, but if we do and it's some kind of free-trier one as a dummy, then there should be a comment explaining that.

I doubt the token above works anymore, but would be interesting to check. And if it does it might be worth telling Grove people to disable it.

Reply:
> Agree. But I'm curious how we handle the case when we're an open-source project and people try to get the token via intercepting network requests or decompiling?

I have the same question as you @qfrank. It's a good one to ask in Discord as well.

Reply:
I have opened a ticket with Grove support to disable this token.